packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -420,6 +476,16 @@ class HorizontalOverflow extends Foundation<
         for (const item of items) {
             itemWidths.push(getClientRectWithMargin(item).width);
         }
+        return { availableWidth, itemWidths };
+    }
+
+    /**
+     * Handler for the click event fired after next or previous has been clicked
+     */
+    private handleClick(direction: ButtonDirection): void {
+        const regionWidthInformation: any = this.getItemsWidthAndTotalWidth();
```

Comment:
this should not be any.

Reply:
See above.

Reply:
+1 @nalogiudice if we create an interface for the return type above we can use it here as well